Reply/app/src/main/java/com/example/reply/ui/ReplyHomeViewModel.kt

```diff
@@ -50,7 +54,7 @@ class ReplyHomeViewModel(private val emailsRepository: EmailsRepository = Emails
 }
 
 data class ReplyHomeUIState(
-    val emails : List<Email> = emptyList(),
+    val emails: List<Email> = emptyList(),
```

Comment:
Maybe you need to integrate spotless?

Reply:
I plan to add it and also CI pipeline like other samples i think once it's merged so it's easily tested.